components/InputScanner.js

```diff
@@ -33,12 +32,12 @@ class InputScanner extends React.Component {
       <View style={styles.container}>
         {this.state.hasCameraPermission === null ?
           <Text>Requesting for camera permission</Text> :
-          this.state.hasCameraPermission === false ?
-            <Text>Camera permission is not granted</Text> :
-            <BarCodeScanner
+        this.state.hasCameraPermission === false ?
+          <Text>Camera permission is not granted</Text> :
+          <BarCodeScanner
               onBarCodeRead={this._handleBarCodeRead}
               style={{ height: 200, width: 200 }}
-            />
```

Comment:
Why do these have weird block next to them in code review?

Reply:
@Din-s1 The ones which if you click on it they expand? They're just code that hasn't been changed and is hidden for space :)